rhtconsulting/rhc-ose
# Updated install tools to set the routingConfig/subdomain

#### What does this PR do?

Updated install tools to set the routingConfig/subdomain as well as adding support for setting the subdomain through the configuration file. The default is "apps", but it can now be set to a chosen value instead.
#### How should this be manually tested?

Perform an install, and observe the setting in /etc/openshift/master/master-config.yaml --> routingConfig/subdomain be set to the desired value. Also check the DNS configuration (on the local DNS server) and observe that the wildcard entry uses the same subdomain.
#### Is there a relevant Issue open for this?

Yes, issue #44 
#### Who would you like to review this?

/cc @etsauer
